src/main/groovy/FizzBuzz.groovy

```diff
@@ -0,0 +1,13 @@
+
+def fizzBuzz(n) {
+  //ここに素数を一文字づつ数列化した際のn番目の数値を出力するプログラムを実装してください。
+  //BEGIN_CHALLENGE
+
+  result = ""
```

Comment:
I think this code is confusing.

This result var is not necessary.
The reason why this fizzBuzz function returns correct value is combination of following.

- `result = "hoge"` returns "hoge"
- if statement returns value
- `return` is able to omit